rocket-chat-android-widgets/src/main/res/values/message_styles.xml

```diff
@@ -17,6 +17,9 @@
 
     <style name="TextAppearance.RocketChat.MessageAttachment.Title" parent="TextAppearance.AppCompat.Title">
         <item name="android:textSize">14sp</item>
+    </style>
+
+    <style name="TextAppearance.RocketChat.MessageAttachment.Title.Link" parent="TextAppearance.RocketChat.MessageAttachment.Title">
```

Comment:
`parent="TextAppearance.RocketChat.MessageAttachment.Title"` is not needed.

The default parent of a style `AAA.BBB.CCC` is `AAA.BBB`.